operations/dd-toc-guide.md

```diff
@@ -49,6 +55,57 @@ We expect all TOC members to be mindful of their obligations and timelines, whet
 
 The issue will contain a limited set of information about the project at the time of its application, commonly asserting its conformance to the stated criteria with links to where or descriptions as to how they are implemented.
 
+### Initial triage/evaluation prior to assignment
+
+All TOC members are expected to assist in the triaging of project applications to move levels to ensure that when a TOC member is ready to be assigned, the project is ready to be evaluated with no immediate blockers that would inhibit or delay the TOC's engagement.
+
+This light-weight triage/evaluation must cover the list below (it is not exhaustive and is a minimum triage set from the [incubation template retrieved 12 DEC 2025](https://github.com/cncf/toc/blob/c2943ffc98064dd88e9ef9c4afd5a8856898942f/.github/ISSUE_TEMPLATE/template-incubation-application.md)):
```

Comment:
To reduce TOC workload, I'd be interested in having projects check they have done these before submitting the move level req.  That would save TOC from spending time evaluating many projects that are not ready.

Reply:
The application templates are designed to be that check list for completing the criteria and includes these items as checkboxes. The check boxes in the template are intended to be the assertion by the project of their completion of those, however projects are not completing all of them or are marking some as complete but may not have completed them. 

The exception to this is the recent changes that include the adopter form to collect those 5-7, and the explicit call out of 5-7 (i believe this only exists in our guide)  because we usually only get responses from 3, maybe 4. 

I think we can do better highlighting the joint assessment and governance reviews in the application like you suggested, we just need to make sure we're clear these are not required but definitely are encouraged to be completed as their contents cover those sections.  Let's get that captured in an issue - Done in https://github.com/cncf/toc/issues/1511

Reply:
I've ensured this is also captured in the README of the process for projects